src/Exception/ForbiddenException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Exception;
+
+/**
+ * @internal
+ */
+final class ForbiddenException extends AbstractApiException
+{
+    public function __construct(string $message = 'Access Denied')
+    {
+        parent::__construct(403, $message);
```

Comment:
We should use the HttpResponseCodes enum here. I already changed it for the openapi responses, maybe we should use them in the exceptions too? WDYT?